packages/ui-extensions/src/surfaces/point-of-sale/components/NumberField/NumberField.ts

```diff
@@ -5,13 +5,11 @@ import type {InputProps} from '../shared/InputField';
  * Represents the properties for the NumberField component.
  * @typedef {Object} NumberFieldProps
  * @property {'decimal' | 'numeric'} [inputMode] - The mode of input, can be either 'decimal' or 'numeric'.
- * @property {number} [max] - The highest decimal or integer to be accepted for the field.
- * @property {number} [min] - The lowest decimal or integer to be accepted for the field.
  */
 export interface NumberFieldProps extends InputProps {
-  inputMode?: 'decimal' | 'numeric';
```

Comment:
Sorry I didn't mean to say comment these out. I meant to not have comments here! We can't comment these out because that would be a breaking change to developers.